pkg/daemon/update.go

```diff
@@ -1725,8 +1725,28 @@ func getFileOwnership(file ign3types.File) (int, int, error) {
 }
 
 func (dn *Daemon) atomicallyWriteSSHKey(keys string) error {
-	authKeyPath := filepath.Join(coreUserSSHPath, "authorized_keys")
+	var authKeyPath string
 
+	// If we are using FCOS then use /home/core/.ssh/authorized_keys.d/ignition
+    if dn.os.IsFCOS() {
+		authKeyPath = filepath.Join(coreUserSSHPath, "authorized_keys.d/authorized_keys_slice")
+		authKeyPathIgnite := filepath.Join(coreUserSSHPath, "authorized_keys.d/ignition")
+		authKeyPathDefault := filepath.Join(coreUserSSHPath, "authorized_keys")
+
+		//First time rename authorized_keys.d/ignition to authorized_keys.d/authorized_keys_slice
```

Comment:
Why not use `/home/core/.ssh/authorized_keys`?

Reply:
To me that defeats the purpose of the PR which is to enable authorized_keys.d/ usage.  This negates the need for Christian's security PR, so ssh works out of the box.  I do wish that the ignition process hadn't named the file `ignition` though.  

Although I do see the alternative to use authorized_keys and just delete authorized_keys.d.  It would simplify the code.

I'm not passionate around either as long as "something: is done...